gradle-plugin/kubernetes/src/main/java/org/eclipse/jkube/gradle/plugin/KubernetesPlugin.java

```diff
@@ -45,6 +46,7 @@ protected void jKubeApply(Project project) {
     register(project, "k8sBuild", KubernetesBuildTask.class);
     register(project, "k8sResource", KubernetesResourceTask.class);
     register(project, "k8sApply", KubernetesApplyTask.class);
+    register(project, "k8sLog", KubernetesLogTask.class);
```

Comment:
Should be added to the OpenShiftPlugin class too
```java
register(project, "ocLog", KubernetesLogTask.class);
```

Reply:
Umm, I thought I would do it when I'll add OpenShiftLogTask

Reply:
For OpenShiftLogTask we'll revisit this, but so far OpenShift needs to behave, at least, like Kubernetes. Since OpenShiftPlugin doesn't extend KubernetesPlugin, we need to add tasks there too. If a Task needs specialization,it can be overridden and replaced with the appropriate OpenShiftXXXTask